extract/extract.go

```diff
@@ -436,10 +436,12 @@ func GetMinor(part string) string {
 	return minor
 }
 
+const defaultMinorVersion = 15
+
 func genBuildTags() (string, error) {
 	version := runtime.Version()
-	if version == "devel" {
-		return "", nil
+	if strings.HasPrefix(version, "devel") {
+		return fmt.Sprintf("1.%d", defaultMinorVersion), nil
```

Comment:
actually we might not want to do that, I'll think about it again

Reply:
I think that this change is not necessary. As maintainers, we want to run `extract` only for stable Go versions. By removing the constraint on next version, it is still possible to re-build yaegi with next and have a working stdlib.

May be add a comment in the Makefile to warn against the necessity to run `go generate` on with an `devel` version of Go compiler (but not forbid to do so).

Reply:
I know that the extract/ lib can be used by other projects, and not just for extracting the stdlib for yaegi, so it might make sense to allow "devel", but at the moment genBuildTags is actually only called in the case of the stdlib, for which we do not want to allow "devel" at the moment. So I think it's legitimate to return an error for now. And it makes the failure explicit when someone tries to run "make generate" with it. We can always change it back to be more lenient later, if we ever call genBuildTags for other things. WDYT?

Reply:
ok